yvalenta/lib/server/connection.rb

```diff
@@ -0,0 +1,35 @@
+require 'securerandom'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.